src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/helpers.ts

```diff
@@ -58,6 +64,15 @@ export function getSubqueriesToValidate(rootCommands: ESQLCommand[]) {
       }
     }
 
+    // every command within FROM's subqueries is its own subquery to be validated
+    if (command.name.toLowerCase() === 'from') {
```

Comment:
I wonder if it makes sense to validate FORK in sub queries

<img width="1775" height="662" alt="image" src="https://github.com/user-attachments/assets/0c371be6-7f90-4519-a112-d34f4c247976" />